roles/jekyll/files/jekyll.conf

```diff
@@ -1,6 +1,6 @@
 <VirtualHost lug.utdallas.edu:80>
 	ServerName lug.utdallas.edu
-	DocumentRoot /var/www/home/
+            Redirect permanent / https://lug.utdallas.edu
```

Comment:
Please fix this indentation, this is not in line with the style of the rest of the repo.